### PR TITLE
P0 41 Button

### PR DESCRIPTION
**Pull request by tomaszdanczak**

### CODE REVIEW

File: src/components/atoms/Button/Button.styles.js
Comment (LachPawel) [inline src/components/atoms/Button/Button.styles.js:12]: Note: Once we merge the branch with typography, you can use the utils here.

File: src/components/atoms/Button/Button.styles.js
Comment (LachPawel) [inline src/components/atoms/Button/Button.styles.js:40]: Note: All of the color values can be taken from the utils once they are merged, just like above. However, for the time being it's OK :)

**Review by LachPawel — APPROVED** [review]
Overall, looks good congratulations :+1:  Especially storybook part is great :) Maybe you can just wait for previous task to get merged, pull the changes and adjust those details.

**Comment by tomaszdanczak** [comment]
Note: I will do some improvements in future (I will use variables and add tests)

**Pull request merged**
